### ISSUE TITLE
tslint:disable-next-line: max-line-length doesn't work before typescript multiline template string

### ISSUE THREAD

**Issue by doradsoft** [Microsoft/vscode-typescript-tslint-plugin#74]
While putting 

> // tslint:disable-next-line: max-line-length

 before a multiline [template string](https://basarat.gitbooks.io/typescript/docs/template-strings.html) block which contains a long line, warning is still presented.

Reproduce:

> // tslint:disable-next-line: max-line-length
> \`\<html\>
> lonnnnnnnnnnnnnnnnnnnnnnnnnnnnnnngggggggggggggggggggggggggggggggggggggggggggg linnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnne
> \<\/html\>\`

**Comment by mjbvz** [comment]
Do you also see this issue when running tslint from the command line?

**Comment by glen-84** [comment]
Use:

```ts
// tslint:disable:max-line-length
`<html>
lonnnnnnnnnnnnnnnnnnnnnnnnnnnnnnngggggggggggggggggggggggggggggggg linnnnnnnnnnnnnnnnnnnnnnnnnnne
</html>`;
// tslint:enable:max-line-length
```

`disable-next-line` is for a single line.

Regardless, you should report such issues [here](https://github.com/palantir/tslint/issues) in future. This repository is for the VS Code extension, not TSLint itself.

**Comment by mjbvz** [comment]
Closing as external tslint issue
